src/pages/home/ReportScreen.js

```diff
@@ -258,6 +259,12 @@ function ReportScreen({
     const onSubmitComment = useCallback(
         (text) => {
             Report.addComment(getReportID(route), text);
+            // we need to scroll to the bottom of the list after the comment was added
+            const refID = setTimeout(() => {
+                flatListRef.current.scrollToOffset({animated: false, offset: 0});
+            }, 10);
+
+            return () => clearTimeout(refID);
```

Comment:
When a new item is added, it will appear in a new frame. If there's no timeout, the scroll will automatically move to the last message before the new item is added. As a result, the new item might be positioned outside the viewport

Reply:
afaik Rory OOO

It would be nice to have the option to use sync addComment, but it would be out of current patterns.

Reply:
@s77rt , we may open a separate issue if it doesn't look good for you (as it is a default behavior here)

Reply:
Discussing this internally. We may either:
- Keep setTimeout and open new issue?
- Remove setTimeout (keep the bug) and open a new issue

cc @pecanoro

Reply:
Posted in slack, but I think we should:

1. Keep the setTimeout and include a comment, explaining the bug and including a link to the follow-up issue
1. Investigate a better solution in the follow-up issue

Reply:
Potential alternate solution:

1. Make `Report.addComment` return the optimistic `reportActionID` of the new comment synchronously (i.e: not returning a promise, just an ID)
1. When the comment with that ID appears in the `reportActions` prop for this screen, then perform the scroll.

I think this would be inline with our code patterns and would essentially be a workaround way to do `Report.addComment().then(...)`

Reply:
So if that alternate solution works, it's a better plan than keeping the `setTimeout` and creating a follow-up issue – just address the bug with a long-term solution in this PR.

Reply:
@roryabraham I'll be OOO until Friday. I barely fixed the conflicts as the internet here is terrible. I can handle this starting next week, or maybe we can proceed with your first solution involving a separate PR

Reply:
@s77rt, what do you think? Can we handle this PR before I come back from vacation? It would unblock the rest of the guys.

Reply:
@perunt Going.to retest and complete the checklist now. @roryabraham Will handle whether this can be merged